website/content/docs/operations/stateful-workloads.mdx

```diff
@@ -0,0 +1,246 @@
+---
+layout: docs
+page_title: Considerations for Stateful Workloads
+description: |-
+  Learn about persistent storage options for stateful workloads on Nomad.
+---
+# Considerations for Stateful Workloads
+
+By default, HashiCorp Nomad's allocation storage is ephemeral. Nomad can discard
+it during new deployments, when rescheduling jobs, or if it loses a client. This
+is undesirable when running persistent workloads such as databases.
+
+This document explores the options for persistent storage of workloads running
+in Nomad. The information provided is for practitioners familiar with Nomad and
+with a foundational understanding of storage basics. 
+
+At the end of this article, we provide a [storage
+comparison](#storage-comparison) table with the advantages, disadvantages, and
+the ideal use-case for each storage option. This guidance will help you
+understand and choose the best storage option for your Nomad allocation storage.
+
+This document addresses traditional storage you can attach as a filesystem and
+does not cover object storage such as AWS S3 or MinIO. 
```

Comment:
These are reference docs, not articles. We don't need to say things like "at the end of this article" and there's an auto-generated table of contents.